lms/views/api/lti.py

```diff
@@ -28,8 +28,18 @@ def __init__(self, request):
             lis_result_sourcedid=self.parsed_params["lis_result_sourcedid"],
         )
 
+    @view_config(route_name="lti_api.result.record", schema=APIRecordResultSchema)
+    def record_result(self):
+        """Proxy result (grade/score) to LTI Outcomes Result API."""
+
+        self.request.find_service(name="lti_outcomes_client").record_result(
+            self.outcome_request_params, score=self.request.parsed_params["score"]
+        )
```

Comment:
It doesn't seem to be documented in `LTIOutcomesClient` but I believe this call can raise certain exception classes (for example if the request to the LTI outcomes service fails). What happens if this raises?

Reply:
Oooho, good catch. It can raise `lms.services.exception.ExternalRequestError`, which does not have a corresponding error view.

We have a couple of options here. I notice that `ExternalRequestError` has several subclasses (e.g. `HAPIError`, `CanvasAPIError`). We could sub-class `ExternalRequestError` in some way specific to the `LTIOutcomes` stuff, or we could use it directly without sub-classing (and make an error view for it). What think you?

Given that this whole feature is very additive and behind a feature flag, would you mind if I handled this as a separate PR? Promise to do it this week.